modules/prompt/functions/prompt_sorin_setup

```diff
@@ -129,10 +129,14 @@ function prompt_sorin_setup {
   zstyle ':prezto:module:git:info:keys' format \
     'status' '%b %p %c:%s%A%B%S%a%d%m%r%U%u'
 
-
+  # Default is to show return value
+  local show_return="✘ "
+  if zstyle -T ':prezto:module:prompt' show-return-val; then
+    show_return='✘ $? '
```

Comment:
I believe this would be better as `%?` which relies on zsh tracking it for the prompt, rather than a shell variable.

Reply:
Fixed!